packages/terra-layout/tests/jest/Layout.test.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+
+import Layout from '../../src/Layout';
+import { getBreakpointSize, getCustomProps } from '../../src/utilities';
+
+jest.mock('../../src/utilities');
+
+describe('Layout', () => {
+  it('should render a Layout without optional props', () => {
+    getBreakpointSize.mockImplementation(() => 'tiny');
+
+    const result = shallow((
+      <Layout />
+    ));
+    expect(result).toMatchSnapshot();
+  });
+
+  it('should render a Layout with custom props', () => {
+    getBreakpointSize.mockImplementation(() => 'tiny');
+    getCustomProps.mockReturnValueOnce({ 'data-test-attr': 'test-val' });
+
+    const result = shallow((
+      <Layout />
+    ));
+    expect(result).toMatchSnapshot();
+  });
+
+  describe('Menu Disabled', () => {
+    const layout = (
+      <Layout
+        header={<div>Test Header</div>}
+      >
+        <div>Test Content</div>
+      </Layout>
+    );
+
+    it('should render a Layout when tiny', () => {
+      getBreakpointSize.mockImplementation(() => 'tiny');
+
+      const result = shallow(layout);
+      expect(result).toMatchSnapshot();
+    });
+
+    it('should render a Layout when small', () => {
+      getBreakpointSize.mockImplementation(() => 'small');
+
+      const result = shallow(layout);
+      expect(result).toMatchSnapshot();
+    });
+
+    it('should render a Layout when medium', () => {
+      getBreakpointSize.mockImplementation(() => 'medium');
+
+      const result = shallow(layout);
+      expect(result).toMatchSnapshot();
+    });
+
+    it('should render a Layout when large', () => {
+      getBreakpointSize.mockImplementation(() => 'large');
+
+      const result = shallow(layout);
+      expect(result).toMatchSnapshot();
+    });
+
+    it('should render a Layout when huge', () => {
+      getBreakpointSize.mockImplementation(() => 'huge');
+
+      const result = shallow(layout);
+      expect(result).toMatchSnapshot();
+    });
+  });
+
```

Comment:
Should we add some with menuText to confirm the toggled text?

Also, add test when children are not provided.

Reply:
The menuText doesn't show up in the shallow Layout tests; it does show up in the HoverTarget snapshots. I will update one of these tests to provide menuText and at least validate it doesn't blow anything up.

The nightwatch tests also validate the menuText presence, so we should have decent coverage.

Reply:
and the no child case is covered by the first test